Makefile

```diff
@@ -319,7 +319,16 @@ build-minimal:
 build-api:
 	BUILD_GRPC_FOR_BACKEND_LLAMA=true BUILD_API_ONLY=true GO_TAGS=none $(MAKE) build
 
-dist: build
+dist:
+	STATIC=true $(MAKE) backend-assets/grpc/llama-cpp-avx2
+	STATIC=true $(MAKE) backend-assets/grpc/llama-cpp-avx
+	STATIC=true $(MAKE) backend-assets/grpc/llama-cpp-fallback
+ifeq ($(OS),Darwin)
+	$(info ${GREEN}I Skip CUDA build on MacOS${RESET})
+else
+	$(MAKE) backend-assets/grpc/llama-cpp-cuda
+endif
+	$(MAKE) build
```

Comment:
this calls `grpcs` target which calls make on all the llama variants

Reply:
correct, I've split that off initially just for `STATIC=true` in order to build statically the (non-cuda) llama-cpp backends.

I've added the cuda switch here and not at the top because we don't need container images or the generic `build` target to include a specific backend for cuda - container images are already built with the BUILD_TYPE which compiles all the `llama-cpp` variants against the accellerator library for the specific container images (sycl, hipblas, cuda, etc).